src/controllers/controlpickermenu.cpp

```diff
@@ -881,7 +881,7 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
 
         QString assignMenuTitle = tr("Effect Unit Assignment");
         QString assignString = tr("Assign ");
-        QMenu* effectUnitGroups = addSubmenu(assignMenuTitle,
+        QMenu* effectUnitGroups = addSubmenu(std::move(assignMenuTitle),
```

Comment:
This QString is passed finally to `addMenu(const QString &text)`. 
So this warning should be solved by changing addSubmenu to take `const QString&` as well-